Tests/Tests/STPPaymentMethodFunctionalTest.m

```diff
@@ -0,0 +1,92 @@
+//
+//  STPPaymentMethodFunctionalTest.m
+//  StripeiOS Tests
+//
+//  Created by Yuki Tokuhiro on 3/6/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "STPNetworkStubbingTestCase.h"
+
+@import Stripe;
+
+@interface STPPaymentMethodFunctionalTest : STPNetworkStubbingTestCase
+
+@end
+
+@implementation STPPaymentMethodFunctionalTest
+
+- (void)setUp {
+//    self.recordingMode = YES;
+    [super setUp];
+}
+
+- (void)testCreatePaymentMethod {
+    STPAPIClient *client = [[STPAPIClient alloc] initWithPublishableKey:@"pk_test_vOo1umqsYxSrP5UXfOeL3ecm"];
```

Comment:
is this key meant to be hardcoded?

Reply:
Yeah this seems to be the pattern in functional tests.

Reply:
:+1: The key *is* publishable 😀 

IDK if it's as important with the network stubbing functionality that was recently added, but we'd certainly need a key if/when re-recording.